spec/workers/reactions/create_worker_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+
+RSpec.describe Reactions::CreateWorker, type: :worker do
+  describe "#perform" do
+    let(:reactable_type) { "Comment" }
+    let(:user_id) { 8 }
+    let(:reactable_id) { 1 }
+    let(:category) { "like" }
+    let(:worker) { subject }
+
+    context " when no user found" do
```

Comment:
Can you remove the leading space for each of these descriptions?

Reply:
@mstruve Sure. I thought that was better for reading but I'll do the changes.